### ISSUE TITLE
Pressing enter while select is open triggers error in console

### ISSUE THREAD

**Issue by jacobq** [Dogfalo/materialize#6016]
## Current Behavior
See [this codepen example](https://codepen.io/jacobq_/pen/PRzpXX?editors=1010):
1. Click on select (taken from docs site examples)
2. Press "enter" key (without using up/down keys first)
3. Observe the following error in the console:

```
Uncaught TypeError: Cannot read property 'click' of undefined
    at Dropdown._handleDropdownKeydown (materialize.js:2510)
_handleDropdownKeydown @ materialize.js:2510
```

## Expected Behavior
If no item is selected then the behavior I would expect is either to close the select (leaving value/selection unchanged) or do nothing (like it does now except without the error getting thrown).

## Your Environment
* Version used: 1.0.0-rc.2 (see codepen)
* Browser Name and version: tested in Chrome 67
* Operating System and version (desktop or mobile): Debian Linux (kernel 4.9.0-6-amd64)

**Comment by jacobq** [comment]
One possible "solution"/band-aid would be to add checks [here](https://github.com/Dogfalo/materialize/blob/9de0babd3b69dd7ad8f25c18f0a7d83ef5dc1839/js/dropdown.js#L291-L298) to prevent attempting to call `.click()` on `undefined`.

**Comment by acburst** [comment]
Fixed in 1f5deb3c
